### ISSUE TITLE
Banner for incomplete translations

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse-frontend#419]
## Problem

Currently, if a wp.org Rosetta site is only partially translated, especially if the % translated is _low_, like 1, or 2%, or the translation threshold is _high_  but not for key UI elements like the navigation, users might think there's something wrong with the site, their browser, or other issues when they attempt to navigate to a localized site.

## Solution 

We should implement [the banner Learn uses](https://learn.wordpress.org/?locale=dv) to make incomplete translations clear to the user. Learn uses data from the rest api to accomplish this: https://github.com/WordPress/learn/pull/202/files#diff-ddff3385c614ff999459619af1b876da8a6de11f0a5f0a774453db8d9bec1e5bR39-R52

**Comment by obulat** [comment]
I've tried to implement this banner, but I got stuck on implementation details:

The translation status API does not have a CORS header, so it is only possible to get the json from the WordPress.org theme.
I tried to create the following process:
1. Schedule a daily [API call](https://translate.wordpress.org/api/projects/meta/openverse/) from the Openverse WordPress.org child theme and save the result as a json file.
2. With every `emitLocale` in the WP.org Openverse theme, additionally send the translation status info from the json file.
3. Use the `NotificationBanner.vue` component to show the translation percentage for current locale.
But the problem is that I don't know how to do the first part in PHP.
